iphone/Classes/GeolocationModule.m

```diff
@@ -848,6 +854,40 @@ - (void)requestLocationPermissions:(CLAuthorizationStatus)authorizationType with
   }
 }
 
+#if IS_SDK_IOS_14
+- (void)requestTemporaryFullAccuracyAuthorization:(NSString *)purposeKey withCallback:(JSValue *)callback
+{
+  if (![TiUtils isIOSVersionOrGreater:@"14.0"]) {
+    NSMutableDictionary *propertiesDict = [TiUtils dictionaryWithCode:1 message:@"Supported on iOS 14+"];
+    NSArray *invocationArray = [[NSArray alloc] initWithObjects:&propertiesDict count:1];
+    [callback callWithArguments:invocationArray];
```

Comment:
Why not just use the short-hand array syntax (and drop the alloc/init/release for a one-short one-element array)?
```suggestion
    [callback callWithArguments:@[ propertiesDict ]];
```

Reply:
Done. Thanks!